extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -596,10 +605,13 @@ public static function getMemcKey( $params ) {
 	 * @return string markup of svg to be used in template
 	 * or empty string if no badge applicable
 	 */
-	public function getUserBadgeMarkup( $userGroups ) {
+	public function getUserBadge( $userGroups ) {
 		foreach ( self::PERMISSION_HIERARCHY as $group ) {
 			if ( in_array( $group, $userGroups ) ) {
-				return DesignSystemHelper::getSvg( self::PERMISSIONS_TO_BADGES[ $group ] );
+				return [
+					'badgeMarkup' => DesignSystemHelper::getSvg( self::PERMISSIONS_TO_BADGES[ $group ] ),
+					'badgeText' => wfMsg( self::PERMISSIONS_TO_TEXT[ $group ] )
```

Comment:
wfMsg is deprecated. Please use wfMessage and use an explicit ->plain() conversion